app/views/common/confirm_class_clean_up.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= _('Unknown records found in the database') %></h1>
+
+<%= alert(text: _('Creating a DB backup before proceeding is highly recommended!'), class: 'alert-danger', close: false) %>
+
+<%= (_('Foreman has detected definitions of unknown classes in your database. This can happen after a plugin removal which did not properly cleaned up data it created. Would you like to delete all records from SQL table <b>"%{table}"</b> with column <b>"%{column}"</b> having the value <b>"%{value}"</b>') % { table: @parent_class.table_name, column: @parent_class.inheritance_column, value: @unknown_class_name}).html_safe %>
+
+<br><br>
+<%= link_to 'Delete', '?confirm_data_deletion=yes', class: 'btn btn-danger', data: { confirm: 'The data deletion can not be undone, are you sure you want to proceed?' } %>
```

Comment:
Does this mean it's a GET instead of a POST action? That's usually not good for destructive actions.

Reply:
yes, the reason IIRC was we'd have to define new generic action and route because we couldn't reuse the current rescue_from action

however I also noticed this line is missing string extraction, I'll send a fixing PR

Reply:
and to be more specific, button_to does not allow keeping url and just adding `?confirm` url param, which we need for reusing the same rescue method